macaroni-os/kit-fixups
# dev-lang/php

Summary
========
* switched to autogen.yaml
* deleted multiple obsolete templates and patch files

Test Plan
========
* Doit
```
doit
INFO     Autogen: dev-lang/php (latest)                                                                                                                                                                                    
INFO     Download from https://github.com/php/php-src/tarball/0454901ed1518cfb42bc77ac91728591b58c3e6e verified as valid tar.gz archive.                                                                                   
INFO     Created: php-8.4.1.ebuild 
```
* Make it available in a own repo tree or by seeding into a local overlay
* Emerge it
```
workstation-mark-repo ~ # emerge -av php

These are the packages that would be merged, in order:

Calculating dependencies... done!
[ebuild  N     ] dev-lang/php-8.4.1:8.4::overlay  USE="acl argon2 berkdb bzip2 cgi cli ctype curl exif fileinfo filter flatfile fpm gd gdbm iconv intl ipv6 mysql mysqli nls opcache pcntl pdo phar posix readline session simplexml sockets sodium spell sqlite ssl tidy tokenizer truetype unicode webp xml xmlreader xmlwriter xslt zip zlib -apache2 -apparmor -bcmath -calendar -cdb -cjk -coverage -debug -embed -enchant -ffi -firebird -ftp -gmp -imap -inifile -iodbc -kerberos -ldap -ldap-sasl -libedit -libressl -lmdb -mhash -mssql -oci8-instant-client -odbc -phpdbg -postgres -qdbm (-selinux) -session-mm -sharedmem -snmp -soap -sysvipc -test -threads -tokyocabinet -xpm -zts" 0 KiB

Total: 1 package (1 new), Size of downloads: 0 KiB

Would you like to merge these packages? [Yes/No] 
>>> Verifying ebuild manifests
>>> Emerging (1 of 1) dev-lang/php-8.4.1::overlay
>>> Installing (1 of 1) dev-lang/php-8.4.1::overlay
>>> Recording dev-lang/php in "world" favorites file...
>>> Jobs: 1 of 1 complete                           Load avg: 5.00, 7.52, 5.73

 * Messages for package dev-lang/php-8.4.1:

 * Installing php.ini for cli into /etc/php/cli-php8.4
 * 
 * Adding opcache to /etc/php/cli-php8.4/ext
 * Installing php.ini for cgi into /etc/php/cgi-php8.4
 * 
 * Adding opcache to /etc/php/cgi-php8.4/ext
 * Installing php.ini for fpm into /etc/php/fpm-php8.4
 * 
 * Adding opcache to /etc/php/fpm-php8.4/ext
 * To build extensions for this version of PHP, you will need to
 * add php8-4 to your PHP_TARGETS USE_EXPAND variable.
 * 
 * For details on how version slotting works, please see
 * the wiki:
 * 
 *   https://wiki.gentoo.org/wiki/PHP
 * 
>>> Auto-cleaning packages...

>>> No outdated packages were found on your system.

 * GNU info directory index is up-to-date.
workstation-mark-repo ~ # 

```
* Running it
```
workstation-mark-repo ~ # php -version
PHP 8.4.1 (cli) (built: Nov 24 2024 15:41:13) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.4.1, Copyright (c) Zend Technologies
    with Zend OPcache v8.4.1, Copyright (c), by Zend Technologies
workstation-mark-repo ~ # 

```
